conda/base/context.py

```diff
@@ -487,6 +496,19 @@ def plugin_manager(self):
 
         return get_plugin_manager()
 
+    @property
+    def no_plugins(self):
+        """
+        This property is used to determine if plugins should be loaded or not.  It is set by
+        the ``--no-plugins`` command line flag.  It can also be set by the ``NO_PLUGINS``
+        environment variable.  If neither of those are set, then the default value is used.
+        """
+        return (
+            self._argparse_args
+            and "no_plugins" in self._argparse_args
+            and self._argparse_args["no_plugins"] is not None
+        ) or os.getenv("NO_PLUGINS") is not None
+
```

Comment:
We don't need to define a new `NO_PLUGINS` envvar, `CONDA_NO_PLUGINS` is a freebee we get by adding `_no_plugins = ParameterLoader(...)` to the context.

Reply:
Oh that's interesting!